src/main/Video.tsx

```diff
@@ -58,8 +59,12 @@ const Video: React.FC<{}> = () => {
       } else {
         dispatch(setError({error: true, errorTitle: undefined, errorMessage: t("video.comError-text"), errorDetails: error, errorIcon: undefined}))
       }
+    } else if (videoURLStatus === 'success') {
+      if (!duration) {
+        dispatch(setError({error: true, errorTitle: undefined, errorMessage: t("durationError-text"), errorDetails: error, errorIcon: undefined}))
```

Comment:
Just an unrelated note: In the `interface error`, you could change `errorTitle: string | undefined` to `errorTitle?: string`. And then you don't have to write out `errorTitle: undefined` in objects you create, you can just omit that part then. Same goes for the two other potentially undefined fields.